packages/markdownviewer/src/widget.ts

```diff
@@ -48,7 +48,10 @@ class MarkdownViewer extends Widget {
   constructor(context: DocumentRegistry.Context, rendermime: RenderMime) {
     super();
     this.addClass(MD_CLASS);
-    this.layout = new PanelLayout();
+    let layout = this.layout = new PanelLayout();
+    let toolbar = new Widget();
```

Comment:
The other ones don't have a panel object with children to add the toolbar to. In those cases the widget in the dock panel is the actual content, rather than wrapper around it. I could use a `::before` element for the markdown one, but it messed up the position of the browsers scrolling when a user clicks on an internal `#` reference (it would be under the micro-toolbar.

Eventually (post-beta) we are going to add toolbar/panels to all widgets, but we want to wait until we have TSC based components for the toolbar refactor.